source/billing_system.html.md.erb

```diff
@@ -0,0 +1,74 @@
+# The GOV.UK PaaS billing system
+
+The GOV.UK Platform as a Service (PaaS) billing system generates bills for GOV.UK PaaS tenants. Our tenants use GOV.UK PaaS to create and delete various services and resources, such as compute instances running applications and databases. We need to calculate how much these services and resources cost on a monthly basis so we can recoup this money from tenants.
+
+The GOV.UK PaaS billing system comprises:
+
+- 2 applications written in Golang: the `paas-billing-collector` and the `paas-billing-api`
+- the GOV.UK PaaS billing Postgres database application, which uses stored functions to calculate tenants' bills in the database
+
+## Accessing GOV.UK PaaS billing
+
+To access GOV.UK PaaS billing information, use the PaaS Admin web application through the *Explore your costs and usage* link on your organisation’s page. You can also access more detailed information, such as resource IDs and charging plan details, through the [PaaS Billing API](https://github.com/alphagov/paas-billing/README.md).
+
+PaaS Admin calls the `\billable_events` endpoint of the API. The API then queries the billing database to get the required data and calls a stored function. For example, [the `get_tenant_bill` stored function](https://github.com/alphagov/paas-billing/blob/main/billing-db/sprocs/get_tenant_bill.sql)) is the database entry point for calculating tenant bills.
+
+
```

Comment:
Mention something about admin level stuff found in https://admin.london.cloud.service.gov.uk/platform-admin ?